tools/nebula-algorithm/src/main/scala/com/vesoft/nebula/tools/algorithm/config/NebulaConfig.scala

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package com.vesoft.nebula.tools.algorithm.config
+
+case class NebulaConfig(hostPorts: String,
+                        nameSpace: String,
+                        partitionNumber: String,
+                        labels: List[String],
+                        hasWeight: Boolean,
+                        weightCols: List[String])
+
+object NebulaConfig {
+  var hostPorts: String        = _
```

Comment:
why mutable variables?

Reply:
All the mutable variables in object NebulaConfig can be removed, I will remove them.